web/packages/dmt-core/src/components/UploadFileButton.tsx

```diff
@@ -95,7 +95,7 @@ export function UploadFileButton(props: {
         ref={textInput}
         accept={fileSuffix.map((a: string) => '.' + a).join(',')}
         style={{ display: 'none' }}
-        onChange={(event: any) => handleUpload(event)}
+        onChange={(event) => handleUpload(event)}
```

Comment:
```suggestion
        onChange={(event: Event) => handleUpload(event)}
```
Could we use the [`Event` type](https://developer.mozilla.org/en-US/docs/Web/API/Event) here?